internal/pkg/scheduler/scheduler.go

```diff
@@ -78,6 +87,9 @@ func New(schedules []Schedule, opts ...OptFunc) (*Scheduler, error) {
 	return s, nil
 }
 
+// Run executes all scheduled function according to their schedules.
+// Schedule Interval times are garunteed minium values (if a execution takes a very long time, the scheduler will wait Interval before running the function again).
+// It is a blocking operation.
```

Comment:
Not sure it's worth repeating for every Run method that it is blocking. Every function is blocking by default.

Reply:
removed for blocking calls